include/adaflo/sharp_interface_util.h

```diff
@@ -941,9 +969,16 @@ compute_force_vector_sharp_interface(const Quadrature<dim - 1> &surface_quad,
   std::vector<double>                  buffer_dim;
```

Comment:
While pre-existing, I think this variable would also be more self-explanatory by the context it is used for, which is the normal vector.